src/main/scala/machines/Internal.scala

```diff
@@ -3,4 +3,31 @@ package machines
 import regex._
 import dfa._
 
-// TODO: Add your code below
+given Conversion[Char, RegularLanguage] =
+    c => Character(c)
+
+given Conversion[String, RegularLanguage] = 
+    s => s.map(c => Character(c)).reduce(Concat(_,_))
+
+given Conversion[RegularLanguage, DFA] =
+    l => regexToDFA(l, findCharacters(l))
```

Comment:
I implemented this differently! I used the toDFA extension we coded, which was a little tricky. This way looks a little cleaner than mine.